arcsi/api/show.py

```diff
@@ -389,29 +389,28 @@ def view_show_archive(show_slug):
 @auth_token_required
 def view_show_page(show_slug):
     do = DoArchive()
+    filterArray = {}
```

Comment:
small syntactic nitpick: 
 - you've used camelcase for these vars but please always use the pythonic schema: `filter_array`
 - An `array` is a list `[]` this is a `dict`. Could be `filter_dict`?

Here you may not even need that second part and just call it `filter`.

Reply:
yeppp, thanks for the review, I could simply use the filter_list here, I don't have to create another variable